### PR TITLE
Add triggerView action to Adobe Target web

### PR DESCRIPTION
**Pull request by marinhero**
This PR adds support for the `page()` call in the Adobe Target Web destination via the `triggerView` action.

## Testing

Subscriptions & Settings to test this and the rest of the Target Actions:

```
{
  "client_code": "YourClientCode",
  "admin_number": "YourNumber",
  "version": "2.8.0",
  "cookie_domain": "localhost",
  "mbox_name": "target-global-mbox",
  "subscriptions": [
{
      "partnerAction": "triggerView",
      "name": "Trigger View",
      "enabled": true,
      "subscribe": "type = \"page\"",
      "mapping": {
        "viewName": { "@path": "$.name"},
        "pageParameters": {"@path": "$.properties"},
        "sendNotification": true,
        "traits": {
          "@path": "$.traits"
        }
      }
    },
    {
      "partnerAction": "upsertProfile",
      "name": "Upsert Profile",
      "enabled": true,
      "subscribe": "type = \"identify\"",
      "mapping": {
        "userId": {
          "@if": {
            "exists": {
              "@path": "$.userId"
            },
            "then": {
              "@path": "$.userId"
            },
            "else": {
              "@path": "$.anonymousId"
            }
          }
        },
        "traits": {
          "@path": "$.traits"
        }
      }
    },
    {
      "partnerAction": "trackEvent",
      "name": "Track Event",
      "enabled": true,
      "subscribe": "type = \"track\"",
      "mapping": {
        "userId": {
          "@if": {
            "exists": {
              "@path": "$.userId"
            },
            "then": {
              "@path": "$.userId"
            },
            "else": {
              "@path": "$.anonymousId"
            }
          }
        },
        "type": {
          "@path": "$.event"
        },
        "properties": {
          "@path": "$.properties"
        }
      }
    }
  ]
}
```

Payload

```
{
 "userId": "ELMARLE42",
 "name": "theTestSuite",
 "properties": {
   "favorite_color": "Blue",
   "favorite_icecream": "Rocky Road"
 },
 "traits": {
    "favorite_music": "JAZZ"
  },
 "options": {}
}
```

- [x] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/browser-destinations/src/destinations/adobe-target/triggerView/index.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    sendNotification: {
+      type: 'boolean',
+      description:
+        'By default, notifications are sent to the Adobe Target backend for incrementing impression count.  If false, notifications are not sent for incrementing impression count. ',
+      label: 'Send Notifications to Adobe Target.',
+      default: true
+    }
+  },
+  perform: (Adobe, event) => {
+    const pageName = event.payload.viewName
```
Comment (nick-Ag) [inline packages/browser-destinations/src/destinations/adobe-target/triggerView/index.ts:38]: I think you could get rid of these `const` assignments and just pass values into the functions

**Comment by nick-Ag** [comment]
Curious if there is a stage deploy process doc somewhere? I couldn't find one in our readme. I think I recall you stage testing a previous Adobe PR but did you stage this one?

**Comment by marinhero** [comment]
@nick-Ag I test this using the built-in web server provided by `yarn dev` plus the unit tests included. Haven't gotten to test with staging yet but I'll update the README once I find a flow that works for that.

File: packages/browser-destinations/src/destinations/adobe-target/triggerView/index.ts (hunk 1 above)
Comment (marinhero) [reply packages/browser-destinations/src/destinations/adobe-target/triggerView/index.ts]: good call, the `viewName` key is descriptive enough.

**Review by nick-Ag — APPROVED** [review]

**Pull request merged**